app/scripts/modules/core/src/presentation/flex-layout.less

```diff
@@ -34,6 +34,12 @@
   &.right {
     justify-content: flex-end;
   }
+
+  &.margin-between {
+    > * + * {
```

Comment:
This looks like something Q*bert says when he dies.

Reply:
![6a00d83451c29169e20134817dd4df970c](https://user-images.githubusercontent.com/2053478/30729533-28471970-9f15-11e7-9ce8-76f2d7fbb6c4.jpg)